src/components/NewsRoomCard.jsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  Button
+} from "@baltimorecounty/dotgov-components";
+import React from "react";
+
+const NewsRoomCard = props => {
+  const { publishDate, title, author, articleSummary, thumbnail, url } = props;
+
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric"
+  };
+
+  const published = new Date(publishDate).toLocaleDateString("en-US", options);
+
+  return (
+    <Card>
+      <CardContent>
+        <div style={{ textAlign: "left" }}>
```

Comment:
See the component library for alignment alignment https://baltimorecounty.github.io/dotgov-components/#card

Reply:
updated card to use "text-left" class